_garden/shrine/asymmetric-spaceships/pulp-scifi-book-covers.md

```
@@ -0,0 +1,74 @@
+  - books
+  - sci-fi
+  - spaceship
+---
+
+{%include webring.html %}
+
+The 1970s and 1980s were a golden age of pulp sci-fi book covers, with vibrant, varied designs. I could dedicate a whole page to the genre, but for now let's just focus on some of the asymmetric ones.
+
+## Blas Gallego
+
+More of a [lusty sword-and-sorcery artist](), Gallego seems to have two-ish sci-fi book covers, and the one that's got asymmetric ships happens to be for a book I own!
```

markdownlint-cli2 (0.22.1)

[warning] No empty links

(MD042, no-empty-links)